Packs/CoreAlertFields/IncidentFields/incidentfield-osparentcmd.json

```diff
@@ -28,5 +28,16 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
+<<<<<<< HEAD
+	"x2_fields": "os_actor_process_command_line",
+    "Aliases": [
+        {
+          "cliName": "parentcmdline",
```

Comment:
Parent cmd line should go to CGO CMD (causality_actor_process_command_line)

Reply:
fixed